src/components/internals/wrapper-with-icon/wrapper-with-icon.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import { customProperties } from '@commercetools-uikit/design-system';
+import { components as defaultComponents } from 'react-select';
+
+const getDefaultComponent = (type) => {
+  if (type === 'singleValue') return defaultComponents.SingleValue;
+  if (type === 'placeholder') return defaultComponents.Placeholder;
+  return React.Fragment;
+};
+
+const WrapperWithIcon = ({ children, ...props }) => {
+  const Component = getDefaultComponent(props.type);
+
+  return (
+    <React.Fragment>
+      {props.selectProps.iconLeft &&
+        React.cloneElement(props.selectProps.iconLeft, {
+          size: 'big',
+        })}
+      <span
+        css={css`
+          margin: ${customProperties.spacingXl + customProperties.spacingXs};
```

Comment:
Does it mean placement is sort of out of rhythm of our spacings in general?

Reply:
Kinda, but it's because `react-select` has it's own layout system and uses absolute positioning to place the value inside the input.
The Icon always has 24px (= SpacingsXL), and we want a margin of `spacingsXs` between the icon and the label.

Perhaps I should just use `24px` and add a comment, to be clear?

Reply:
I think the math is fine. Maybe just add a comment why it's needed :)